config/database.yml

```diff
@@ -21,7 +21,7 @@ defaults: &defaults
   checkout_timeout: 5
   reaping_frequency: 10
   variables:
-    statement_timeout: 2500 # ms
+    statement_timeout: <%= Figaro.env.database_statement_timeout.to_i %>
```

Comment:
Would it make sense to add validation somewhere that `database_statement_timeout` is present? Looks like if it's `nil` this will set the timeout to `0`, not sure what effect that will have.

Reply:
Yep. Will add it to `figaro.rb`.